redisinsight/templates/_helpers.tpl

```diff
@@ -0,0 +1,72 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
**question:** Should we keep this line?

Reply:
Me neither, sorry for letting it through. Removing.